src/components/Detail/Memo.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+import colors from '../../styles/Colors';
+import font from '../../styles/Font';
+
+function Memo() {
+  return (
+    <StyledItemWrapper>
+      <StyledItemInWrapper placeholder="메모장"></StyledItemInWrapper>
+    </StyledItemWrapper>
+  );
+}
+
+export default Memo;
+
+const StyledItemWrapper = styled.form`
+  position: relative;
+  margin: 1.5rem 0;
+`;
+
+const StyledItemInWrapper = styled.textarea`
```

Comment:
```suggestion
const StyledTextarea = styled.textarea`
```

이런 식으로 이름을 짓는게 더 파악하기 쉽지 않을까요??

Reply:
이름 지을 때 유심히 생각해봐야겠네요 ..